torch/_ops.py

```diff
@@ -1173,6 +1289,10 @@ def _call_overload_packet_from_python(op: OpOverloadPacket, args, kwargs):
     if torch_function_called:
         return ret
 
+    if _has_pytree_type_in_args_or_kwargs(args, kwargs):
+        op_overload = getattr(op, op.overloads()[0])
+        return op_overload(*args, **kwargs)
```

Comment:
1) this is probably slow and needs caching on the OpOverload
2) this is wrong if someone tries to register multiple OpOverloads for one OpOverloadPacket, so we should error out in that situation